meetings/tool/src/main/resources/main_es.properties

```diff
@@ -0,0 +1,10 @@
+create_new_meeting=Crear nueva reuni\u00F3n
```

Comment:
Please provide _CA and _EU files for this.

Reply:
done